addons/stock_account/models/product.py

```diff
@@ -148,25 +148,24 @@ def _compute_value(self):
                 at_date = at_date.replace(hour=23, minute=59, second=59)
                 product = product.with_context(at_date=at_date)
             valuated_product = product.sudo(False)._with_valuation_context()
-            qty_available = valuated_product.qty_available
-            total_qty_available = valuated_product.with_context(warehouse_id=False).qty_available if self.env.context.get('warehouse_id') else qty_available
+            qty_valued = valuated_product.qty_available
```

Comment:
Not because of this PR but we get the consignment quantity here

Reply:
valuated_product has already a context to remove consignement